WalletWasabi.Gui/Tabs/LegalDocumentsView.xaml

```diff
@@ -3,16 +3,41 @@
              xmlns:controls="clr-namespace:WalletWasabi.Gui.Controls;assembly=WalletWasabi.Gui"
              xmlns:converters="clr-namespace:WalletWasabi.Gui.Converters;assembly=WalletWasabi.Gui"
              x:Class="WalletWasabi.Gui.Tabs.LegalDocumentsView">
-  <DockPanel LastChildFill="True" Background="{DynamicResource ThemeBackgroundBrush}">
-    <TextBlock IsVisible="{Binding EmptyContent}" Text="Text document is missing!" />
-
-    <DockPanel LastChildFill="True" IsVisible="{Binding !EmptyContent}">
-      <Button Command="{Binding AgreeClicked}" IsVisible="{Binding !IsAgreed}" DockPanel.Dock="Top" Margin="15 30 15 10" >I Agree</Button>
-      <controls:GroupBox DockPanel.Dock="Bottom" BorderThickness="0" Classes="docTabContainer" Margin="0">
-        <Grid Classes="content">
-          <controls:MultiTextBox IsReadOnly="True" IsSelectable="False" Text="{Binding Text}" Margin="10 0" TextWrapping="Wrap" AcceptsReturn="True" />
-        </Grid>
-      </controls:GroupBox>
+  <UserControl.Styles>
+    <Style Selector="controls|MultiTextBox">
+      <Setter Property="BorderBrush" Value="{DynamicResource ThemeForegroundBrush}"/>
+      <Setter Property="IsReadOnly" Value="True"/>
+      <Setter Property="IsSelectable" Value="False"/>
+      <Setter Property="TextWrapping" Value="Wrap"/>
+      <Setter Property="AcceptsReturn" Value="True"/>
+    </Style>
+    <Style Selector="controls|MultiTextBox:pointerover /template/ Border#border">
```

Comment:
Can you elaborate on this why are these new Style setters needed?

Reply:
ahh this style in particular is for fixing the look of the border of the legal text (i.e., doesnt changes the border color when focused etc,)

Reply:
ACK! thx.